From cc7ad16ddc8b00442f513f2c2807dbcb14c1e2f3 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 10 Apr 2024 13:45:33 +0200
Subject: [PATCH 0804/1695] plat-stm32mp1: stm32mp1_pwr: fix compatible

Remove the unexpected comma in compatible name "st,stm32mp1,pwr-reg"

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 44024d3df69e4933404096f039df9b8b312635d8 plat-stm32mp1: stm32mp1_pwr: fix compatible]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I72f0b937526789c7a4f46604865822f3d2cb3271
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372729
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp131.dtsi                             | 2 +-
 core/arch/arm/dts/stm32mp151.dtsi                             | 2 +-
 core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c            | 2 +-
 core/arch/arm/plat-stm32mp1/pm/power_config.c                 | 2 +-
 .../devicetree/bindings/regulator/st,stm32mp1-pwr-reg.yaml    | 4 ++--
 5 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index ec5b2614a..cbefcc856 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -259,7 +259,7 @@
 		};
 
 		pwr_regulators: pwr@50001000 {
-			compatible = "st,stm32mp1,pwr-reg";
+			compatible = "st,stm32mp1-pwr-reg";
 			reg = <0x50001000 0x10>;
 
 			reg11: reg11 {
diff --git a/core/arch/arm/dts/stm32mp151.dtsi b/core/arch/arm/dts/stm32mp151.dtsi
index cb0f16c0d..b4aee1d19 100644
--- a/core/arch/arm/dts/stm32mp151.dtsi
+++ b/core/arch/arm/dts/stm32mp151.dtsi
@@ -1205,7 +1205,7 @@
 		};
 
 		pwr_regulators: pwr@50001000 {
-			compatible = "st,stm32mp1,pwr-reg";
+			compatible = "st,stm32mp1-pwr-reg";
 			reg = <0x50001000 0x10>;
 
 			reg11: reg11 {
diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c
index dcc0827fd..f2c003a77 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pwr.c
@@ -295,7 +295,7 @@ static TEE_Result stm32mp1_pwr_regu_probe(const void *fdt, int node,
 }
 
 static const struct dt_device_match stm32mp1_pwr_regu_match_table[] = {
-	{ .compatible = "st,stm32mp1,pwr-reg" },
+	{ .compatible = "st,stm32mp1-pwr-reg" },
 	{ }
 };
 
diff --git a/core/arch/arm/plat-stm32mp1/pm/power_config.c b/core/arch/arm/plat-stm32mp1/pm/power_config.c
index 4a2f2b93a..365f9ac1e 100644
--- a/core/arch/arm/plat-stm32mp1/pm/power_config.c
+++ b/core/arch/arm/plat-stm32mp1/pm/power_config.c
@@ -18,7 +18,7 @@
 #include "context.h"
 #include "power.h"
 
-#define DT_PWR_COMPAT			"st,stm32mp1,pwr-reg"
+#define DT_PWR_COMPAT			"st,stm32mp1-pwr-reg"
 #define SYSTEM_SUSPEND_SUPPORTED_MODES	"system_suspend_supported_soc_modes"
 #define SYSTEM_OFF_MODE			"system_off_soc_mode"
 #define RETRAM_ENABLED			"st,retram-enabled-in-standby-ddr-sr"
diff --git a/documentation/devicetree/bindings/regulator/st,stm32mp1-pwr-reg.yaml b/documentation/devicetree/bindings/regulator/st,stm32mp1-pwr-reg.yaml
index 422e052bf..6b0e57b23 100644
--- a/documentation/devicetree/bindings/regulator/st,stm32mp1-pwr-reg.yaml
+++ b/documentation/devicetree/bindings/regulator/st,stm32mp1-pwr-reg.yaml
@@ -11,7 +11,7 @@ maintainers:
 
 properties:
   compatible:
-    const: st,stm32mp1,pwr-reg
+    const: st,stm32mp1-pwr-reg
 
   reg:
     maxItems: 1
@@ -67,7 +67,7 @@ examples:
   - |
     //At SOC level:
     pwr_regulators: pwr@50001000 {
-      compatible = "st,stm32mp1,pwr-reg";
+      compatible = "st,stm32mp1-pwr-reg";
       reg = <0x50001000 0x10>;
       st,tzcr = <&rcc 0x0 0x1>;
 
-- 
2.47.2

